server/py/services/api/migrations/env.py

```diff
@@ -100,7 +101,10 @@ def run_migrations_online():
                 host=host,
                 db="mlrun",
             )
-            connection.execute(sqlalchemy.sql.text(f"KILL {connection_id};"))
+            try:
+                connection.execute(sqlalchemy.sql.text(f"KILL {connection_id};"))
+            except sqlalchemy.exc.DBAPIError:
```

Comment:
comment out or do error string search on exception to ignore cases when connection is gone?